cmd/kube-scheduler/app/server.go

```diff
@@ -106,6 +110,7 @@ type Options struct {
 // AddFlags adds flags for a specific SchedulerServer to the specified FlagSet
 func (o *Options) AddFlags(fs *pflag.FlagSet) {
 	fs.StringVar(&o.ConfigFile, "config", o.ConfigFile, "The path to the configuration file.")
+	fs.StringVar(&o.WriteConfigTo, "write-config-to", o.WriteConfigTo, "If set, write the default configuration values to this file and exit.")
```

Comment:
If the `config` is not nil, the value is not default, right?

Reply:
No, only write default values from schema, which is my intention.